### PR TITLE
[15.0][MIG] website_sale_hide_price: Migration to 15.0

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT36601

please @chienandalu @sergio-teruel review this

I have added the improvements of this PR https://github.com/OCA/e-commerce/pull/639 and adjusted the changes to version 15.0

### CODE REVIEW

File: website_sale_hide_price/README.rst [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fe--commerce-lightgray.png?logo=github
+    :target: https://github.com/OCA/e-commerce/tree/13.0/website_sale_hide_price
+    :alt: OCA/e-commerce
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/e-commerce-13-0/e-commerce-13-0-website_sale_hide_price
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/113/13.0
```
Comment (sergio-teruel) [inline website_sale_hide_price/README.rst]: Rebuild this file

**Comment by CarlosRoca13** [comment]
@sergio-teruel done

**Review by sergio-teruel — APPROVED** [review]

**Comment by emagdalenaC2i** [comment]
@CarlosRoca13 this PR created a file in other folder.
See https://github.com/OCA/e-commerce/blob/15.0/website_sale_comparison_hide_price/views/website_sale_template.xml

**Comment by pedrobaeza** [comment]
Merged the other module as well, so the problem disappeared: #666

**Pull request merged**